data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -735,11 +737,11 @@ def build_filing_json_meta_data(raw_filing_type: str, filing_type: str, filing_s
                 'fromLegalType': LEGAL_TYPE_CHANGE_FILINGS[event_file_type][0],
                 'toLegalType': LEGAL_TYPE_CHANGE_FILINGS[event_file_type][1],
             }
-        if (old_corp_name := data['old_corp_name']) and (new_corp_name := data['new_corp_name']):
+        if (old_corp_name := data.get('old_corp_name')) and (new_corp_name := data.get('new_corp_name')):
             meta_data['alteration'] = {
                 **meta_data['alteration'],
-                'fromLegalName': old_corp_name,
-                'toLegalName': new_corp_name,
+                'fromLegalName': old_corp_name.replace("'", "''"),
+                'toLegalName': new_corp_name.replace("'", "''"),
```

Comment:
This issue should be fixed via [this formatter](https://github.com/bcgov/lear/blob/7640dee07b22e43d1507cf689f375fb960e0fc01/data-tool/flows/tombstone/tombstone_utils.py#L850-L855) during data loading stage